### PR TITLE
Add TS type for `scrollTo()`

### PR DESCRIPTION
**Pull request by wcandillon**

### CODE REVIEW

File: react-native-reanimated.d.ts [hunk 1]
```diff
@@ -476,6 +476,7 @@ declare module 'react-native-reanimated' {
       pageX: number;
       pageY: number;
     };
+    export function scrollTo(ref: RefOject<ReactNativeScrollView>, x: number, y: number, animated: boolean);
```
Comment (jakub-gonet) [inline react-native-reanimated.d.ts:479]: I believe you have a typo here `RefOject` -> `RefObject`

Comment (wcandillon) [reply react-native-reanimated.d.ts:479]: Thks a lot, that helped me fixed another issue with it. It seems to work great now.

File: react-native-reanimated.d.ts [hunk 2]
```diff
@@ -477,6 +477,8 @@ declare module 'react-native-reanimated' {
       pageY: number;
     };
 
+    export function scrollTo(ref: RefObject<ReactNativeScrollView | ScrollView>, x: number, y: number, animated: boolean);
```
Comment (jakub-gonet) [inline react-native-reanimated.d.ts:480]: Could you please specify return type as well?

```
node_modules/react-native-reanimated/react-native-reanimated.d.ts:479:21 - error TS7010: 'scrollTo', which lacks return-type annotation, implicitly has an 'any' return type.

479     export function scrollTo(ref: RefObject<ReactNativeScrollView | ScrollView>, x: number, y: number, animated: boolean);
                        ~~~~~~~~
```

File: react-native-reanimated.d.ts (hunk 1 above)
Comment (wcandillon) [reply react-native-reanimated.d.ts]: done, thank you :)

**Pull request merged**